openassessment/data.py

```diff
@@ -974,7 +978,7 @@ def create_zip_with_attachments(cls, file, course_id, submission_files_data):
             │   │   │   ├── SomeFile1
             │   │   │   └── SomeFile2
             │   │   ├── part_0.txt
-            │   │   └── part_1.txt
+            │   │  n └── part_1.txt
```

Comment:
Is this intentional?

Reply:
must have been debugging `next` :sweat_smile: